src/views/activity/activity_item_view.js

```diff
@@ -20,5 +26,26 @@ export default BaseView.extend({
 
   sendSelectionDetails (event) {
     webChannel.sendUrlSelected(this.model.url, 'url');
+  },
+
+  injectFavicon () {
+    // Manually construct the favicon element and inject it favicon into the
+    // DOM, allowing us to delegate events to its onLoad event, which is not
+    // possible through Ampersand since load events do not bubble.
+    const favicon = document.createElement('img');
+    favicon.src = this.model.displayImage();
+    favicon.onload = this.calculateIconColor;
+    this.el.querySelectorAll('.favicon')[0].appendChild(favicon);
```

Comment:
If there's only one favicon per activity item view, then I'd just `querySelector('.favicon')`.

But actually, ampersand-view has a nice little something for this case: https://github.com/AmpersandJS/ampersand-view#query-viewqueryclassname